guides/common/modules/con_using-infoblox-as-dhcp-and-dns-providers.adoc

```diff
@@ -4,5 +4,5 @@
 You can use {SmartProxyServer} to connect to your Infoblox application to create and manage DHCP and DNS records, and to reserve IP addresses.
 
 ifdef::satellite[]
```

Comment:
Feels useful to me to include a lower bound for upstream as well. I have no idea how 8.0 was determined though.